src/main/java/mate/jdbc/dao/impl/CarDaoImpl.java

```diff
@@ -0,0 +1,208 @@
+                + " WHERE c.is_deleted = FALSE AND c.id = ?;";
+        Car car = null;
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement statement = connection.prepareStatement(query)) {
+            statement.setLong(1,id);
+            ResultSet resultSet = statement.executeQuery();
+            if (resultSet.next()) {
+                car = parseCarFromResultSet(resultSet);
+            }
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't get car by id: " + id, e);
+        }
+        if (car != null) {
+            car.setDrivers(getDriversForCar(id));
+        }
+        return car;
+    }
+
+    @Override
+    public List<Car> getAll() {
+        String query = "SELECT c.id, c.model, c.manufacturer_id, m.name, m.country"
+                + " FROM cars c"
+                + " JOIN manufacturers m"
+                + " ON c.manufacturer_id = m.id"
+                + " WHERE c.is_deleted = FALSE;";
+
+        List<Car> cars = new ArrayList<>();
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement statement = connection.prepareStatement(query)) {
+            ResultSet resultSet = statement.executeQuery();
+            while (resultSet.next()) {
+                cars.add(parseCarFromResultSet(resultSet));
+            }
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't get all cars", e);
+        }
+        for (Car car : cars) {
+            if (car != null) {
+                car.setDrivers(getDriversForCar(car.getId()));
+            }
+        }
+        return cars;
+    }
+
+    @Override
+    public List<Car> getAllByDriver(Long driverId) {
+        String query = "SELECT c.id, c.model, c.manufacturer_id, m.name, m.country"
+                + " FROM cars c"
+                + " JOIN manufacturers m "
+                + " ON c.manufacturer_id = m.id"
+                + " JOIN cars_drivers cd"
+                + " ON cd.car_id = c.id"
+                + " WHERE cd.driver_id = ?;";
+        List<Car> cars = new ArrayList<>();
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement statement = connection.prepareStatement(query)) {
+            statement.setLong(1,driverId);
+            ResultSet resultSet = statement.executeQuery();
+            while (resultSet.next()) {
+                cars.add(parseCarFromResultSet(resultSet));
+            }
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can get all cars by driverId: " + driverId, e);
+        }
+        for (Car car : cars) {
+            if (car != null) {
+                car.setDrivers(getDriversForCar(car.getId()));
+            }
+        }
+        return cars;
+    }
+
+    @Override
+    public Car update(Car car) {
+        String query = "Update cars SET model = ?, manufacturer_id = ? "
```

Comment:
```suggestion
        String query = "UPDATE cars SET model = ?, manufacturer_id = ? "
```
Fix the same issue in all places